docs/messaging/messaging-spans.md

```diff
@@ -532,19 +534,19 @@ flowchart LR;
 | `messaging.system` | `"rabbitmq"` | `"rabbitmq"` | `"rabbitmq"` |
 | `messaging.destination.name` | `"T"` | `"T"` | `"T"` |
 | `messaging.operation.name` | `"publish"` | `"consume"` | `"consume"` |
-| `messaging.operation.type` | `"publish"` | `"process"` | `"process"` |
+| `messaging.operation.type` | `"send"` | `"process"` | `"process"` |
 | `messaging.message.id` | `"a"` | `"a"`| `"a"` |
 
```

Comment:
In the section "Topic with multiple consumers" we say

> Given is a publisher that publishes a message to a topic exchange

I guess we need to change that to

> Given is a producer that sends a message to a topic exchange

Reply:
Also the example table below it is still using "Publish" terminology
https://github.com/mamwl1/semantic-conventions/blob/mamwl1_branch/docs/messaging/messaging-spans.md#topic-with-multiple-consumers

Reply:
Same comment as for Kafka. messaging.operation.name of publish is appropriate for RabbitMQ.

Reply:
Then should the diagram say `Span Publish A`? Because now it is confusing.. we say publish in the text + in the table but in the diagram is now Send.

Reply:
Before I made any changes:
-the mermaid diagrams use the messaging.operation.type values, in the green boxes, and these values are also used in the table column names
-the descriptive text before the mermaid diagrams used values which are appropriate for the messaging product, and consistent with the span name and messaging.operation.name rows

I tried to stick with that approach when making the change from messaging.operation.type = publish -> send.
It looks like I didn't get this right for the second Kafka example as publish should be send, in the diagram and the column header. I have added that change.

I agree that there is a bit of a disconnect between the different verbs being used. But I think that is inevitable when dealing with many different products. At least we can try and get the approach used across all the examples consistent. 

It also looks like the example at the end of https://opentelemetry.io/docs/specs/semconv/messaging/kafka/ isn't consistent with the approach on the main messaging spans page. I can change that if you agree that the approach above is what we should be using.

And it looks like the "Links" row in the tables are also inconsistent. Sometimes the span name is used, sometimes the column name. Looks like the column name is the one that is generally used, so I can change the values used in https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/#topic-with-multiple-consumers if you are happy with that.

Reply:
Makes sense. We discussed this in the sig meeting and we will see if we need to address this later, in a follow up to not block this. Sorry for may be delaying this.